src/plugins/intel_npu/src/plugin/npuw/base_sync_infer_request.cpp

```diff
@@ -88,66 +88,225 @@ ov::npuw::IBaseInferRequest::RqPtrs ov::npuw::IBaseInferRequest::create_infer_re
             LOG_INFO("Done");
         } else {
-            LOG_INFO("Skip creation of reference subrequest for submodule["
-                     << id << "] on reference device: " << m_npuw_model->m_ref_device << ", as actual subrequest ["
-                     << id << "] has been already created on "
-                     << "it .");
+            LOG_INFO("Skip creation of reference subrequest for Subgraph["
+                    << id << "] on reference device: " << m_npuw_model->m_ref_device << ", as actual subrequest ["
+                    << id << "] has been already created on "
+                    << "it .");
         }
     }
 
     return rqs;
 }
 
-void ov::npuw::IBaseInferRequest::ensure_subrequest_is_accurate(std::size_t idx, bool& failover) {
+namespace {
+    void set_inputs(const ov::SoPtr<ov::IAsyncInferRequest>& from, ov::SoPtr<ov::IAsyncInferRequest>& to) {
+        const auto& from_comp_model = from->get_compiled_model();
+        const auto& to_comp_model = to->get_compiled_model();
+        for (size_t i = 0; i < from_comp_model->inputs().size(); i++) {
+            const auto& itnsr = from->get_tensor(from_comp_model->inputs()[i]);
+            to->set_tensor(to_comp_model->inputs()[i], itnsr);
+        }
+    }
+
+    void copy_results(const ov::SoPtr<ov::IAsyncInferRequest>& from, ov::SoPtr<ov::IAsyncInferRequest>& to) {
+        const auto& from_comp_model = from->get_compiled_model();
+        const auto& to_comp_model = to->get_compiled_model();
+        for (size_t i = 0; i < to_comp_model->outputs().size(); i++) {
+            const auto& from_tnsr = from->get_tensor(from_comp_model->outputs()[i]);
+            const auto& to_tnsr = to->get_tensor(to_comp_model->outputs()[i]);
+            from_tnsr->copy_to(to_tnsr._ptr);
+        }
+    }
+
+    std::stringstream create_launch_msg(std::size_t idx,  std::size_t real_idx) {
+        std::stringstream log_msg_stream;
+        log_msg_stream << "Launching subrequest[" << idx << "]" <<
+        ((real_idx == idx) ? std::string("...").c_str() :
+                             std::string(std::string(", which is actually subrequest[") +
+                                std::to_string(real_idx) + "]").c_str());
+        return log_msg_stream;
+    }
+} // anonymous namespace
+
+void ov::npuw::IBaseInferRequest::try_accurate_subinfer(std::size_t subidx, std::size_t offset,
+                                                        std::size_t len, bool& accuracy_failover) {
+    auto real_subidx = real(subidx);
+    auto& act_subr = m_subrequests.at(real_subidx);
+    if (!m_npuw_model->m_acc_check) {
+        act_subr->infer();
+        return;
+    }
+
+    std::stringstream log_msg_stream = create_launch_msg(subidx, real_subidx);
+    if (m_npuw_model->m_compiled_submodels[real_subidx].spatial && len != 0) {
+        log_msg_stream << ", on range : [" << offset << ", " << offset + len << ")";
+    }
+    log_msg_stream << "...";
```

Comment:
Just write multiple log lines without overdesign.

```
LOG_DEBUG("Your message here");
if (spatial) {
    LOG_BLOCK();
    LOG_DEBUG("Your spatial info here");
}
```

Note - we only write runtime log in DEBUG. It should never be INFO here.